### ISSUE TITLE
Syncthing stops after a second `okteto up` is launched on the same deployment

### ISSUE THREAD

**Issue by rlamana** [okteto/okteto#368]
**Describe the bug**
If a second `okteto up` is launched the first `up` is terminated but syncthing stops working for any consecutive up on the same environment.

**To Reproduce**
1. In a local terminal launch `okteto up`.
2. In another terminal launch `okteto up` for the same environment.
3. Change locally any file.
4. Files are not synched.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 1.3.0

**Additional context**
[okteto.log](https://github.com/okteto/okteto/files/3516908/okteto.log)

**Comment by xiaods** [comment]
i came across same not synced case. even i open one terminal session also can't update in time.

```
OS: macOS 10.14.6
$ okteto version
okteto version 1.4.1 
```

How to debug this situation?

**Comment by pchico83** [comment]
Could you paste the output of `okteto up --loglevel=debug`?
Also, if you want to live chat about the issue, you can join our Kubernetes Slack channel:
https://kubernetes.slack.com/messages/CM1QMQGS0/

**Comment by xiaods** [comment]
with okteto helpdesk's help, it point to syncthing's bug. waiting the tools upgrade then we can check back again.

**Comment by xiaods** [comment]
double check the syncthing v1.2.2 can't sync file again when okteto up update done.
then i edit file again, the syncthing can't sync again.

**Comment by rberrelleza** [comment]
@xiaods could you share the `~/.okteto/okteto.log` and `$HOME/.okteto/*/*/syncthing.log`  to help us troubleshoot the situation?

**Comment by xiaods** [comment]
@rberrelleza i have attach the log.
[okteto.log](https://github.com/okteto/okteto/files/3716744/okteto.log)
[syncthing.log](https://github.com/okteto/okteto/files/3716745/syncthing.log)

```
$ okteto version
okteto version 1.4.9 
```

**Comment by rberrelleza** [comment]
This was fixed by  #502
